tests/Doctrine/Tests/DBAL/Schema/ComparatorTest.php

```diff
@@ -1050,6 +1051,31 @@ public function testCompareChangedColumn(): void
         self::assertEquals($expected, Comparator::compareSchemas($oldSchema, $newSchema));
     }
 
+    public function testCompareChangedColumnLength(): void
+    {
+        $oldSchema = new Schema();
+
+        $tableFoo = $oldSchema->createTable('foo');
+        $tableFoo->addColumn('id', 'string', ['length' => 127]);
```

Comment:
The issue is about BLOB columns. Why does the test compare string column lengths?

Reply:
There was no test yet that tested the length changes for string columns. I mainly added this test to confirm my expectation about how the API works. I can move this test to its own pull request, or remove it alltogehter, however you prefer ☺️